design-proposals/golden-image-delivery-and-update-pipeline.md

```diff
@@ -317,3 +321,30 @@ spec:
       managedDataSource: centos8
 ```
 
+## HyperConverged example
+```yaml
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+spec:
+  featureGates:
+    dataImportCron: true
```

Comment:
based on the previous documentation, should this be `dataImportCron` or `enableCommonDataImportCron`?